src/PSR15/ValidationMiddlewareBuilder.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenAPIValidation\PSR15;
+
+use OpenAPIValidation\PSR7\ValidatorBuilder;
+use Psr\Http\Server\MiddlewareInterface;
+
+class ValidationMiddlewareBuilder extends ValidatorBuilder
```

Comment:
@lezhnev74 I'm totally not sure whether this class deserves it's own builder or we should just union PSR7 and PSR15 namespaces and merge thse builders

Reply:
Good question. Ill review this asap

Reply:
Well, middleware has its own standard (psr15), it does work with psr7 messages, but I think it should stay in its own namespace. I think this decoupling might be good for future expanding.

I vote for keeping both namespaces.

Reply:
If you have doubts, please expand on this. Share your thoughts.

Reply:
In current implementation this middleware is strictly tightened to the PSR-7 validator. As you can see, for now it does nothing more than call request and response validator, so I don't think it would be real independent. The same stands for builder.

Reply:
On the other hand PSR-7 namespace is overflooded too. Schema factory could live somewhere else, as soon it's decopled from builder now. Best option would be `cebe/openapi`, but not sure it would be fast or even happens

Reply:
I think we can improve it later if it goes wrong.